matroid.gemspec

```diff
@@ -30,7 +30,18 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'omniauth-oauth2', '~> 1.3.1'
+  spec.add_dependency 'rest-client', '~> 1.8'
+
   spec.add_development_dependency "bundler", "~> 1.14"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_runtime_dependency 'httparty', "~> 0.14.0"
+  spec.add_development_dependency "minitest"
+  spec.add_development_dependency "vcr"
+  spec.add_development_dependency "fakeweb"
+  spec.add_development_dependency "rspec"
+
+  spec.add_dependency "dotenv"
```

Comment:
can be removed, `ENV` hash is [built-in](https://ruby-doc.org/core-2.2.0/ENV.html)